search/src/Components/wrapper/upload.js

```diff
@@ -20,13 +20,21 @@ const useStyles = (theme) => ({
 })
 
 const baseURL = new URL(window.location.origin);
+const upload_status_mapping = {
```

Comment:
This is unrelated to the present PR, isn't it? Can we pull it into a separate PR?